### PR TITLE
Fix tip division on paper

### PR DESCRIPTION
**Pull request by koutst**
Closes: https://github.com/ResearchHub/issues/issues/143 
- The issue with reward is a formatting confusion caused by the use of a comma as a decimal separator which arises because of the use of toLocaleString.

### CODE REVIEW

File: src/purchase/utils.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
 
 def distribute_support_to_authors(paper, purchase, amount):
     registered_authors = paper.authorship_authors.all()
-    total_author_count = paper.true_author_count()
+    total_author_count = registered_authors.count()
+    print("total_author_count", total_author_count)
```
Comment (yattias) [inline src/purchase/utils.py:11]: remove?

**Review by gzurowski — APPROVED** [review]
LGTM 🚀

**Comment by Carlostt-ux** [comment]
should i reapply for the paper award?

which I tried and it does not allow it

the RSC amount has not changed

should I wait?

**Comment by Carlostt-ux** [comment]
The problem with my account is still persistent, the RSCs corresponding to the award for my paper and the division of the tip have not been credited.

**Pull request merged**